core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -403,6 +437,8 @@ class TransactionCoordinator(brokerId: Int,
   def shutdown() {
     info("Shutting down.")
     isActive.set(false)
+    scheduler.shutdown()
+    txnMarkerPurgatory.shutdown()
```

Comment:
@rajinisivaram also fixed this in the other commit, it's just before the `info` log. So, now we have 2 such calls, we should remove one (considering whether order matters).